SERVING.md

```diff
@@ -0,0 +1,57 @@
+<!---
```

Comment:
Could we somehow avoid spamming out top level dir? This isn't relevant to most people.